javascript/builtins/AggregateError.json

```diff
@@ -88,15 +88,17 @@
             "spec_url": "https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-aggregate-error",
             "support": {
               "chrome": {
-                "version_added": "85"
+                "version_added": false
```

Comment:
Advance note that you probably called the constructor wrong. It needs to be at least `new AggregateError([])`